.github/templates/linux_ci_workflow.yml.in

```diff
@@ -144,8 +144,19 @@ jobs:
       - name: Archive artifacts into zip
         run: |
           zip -r artifacts.zip dist/ build/
+      # Upload to github so that people can click and download artifacts
       - uses: actions/upload-artifact@v2
-        name: Store PyTorch Build Artifacts
+        # Don't fail on upload to GH since it's only for user convenience
+        continue-on-error: true
+        name: Store PyTorch Build Artifacts on Github
+        with:
+          name: ${{ env.BUILD_ENVIRONMENT }}
+          retention-days: 30
```

Comment:
Can we reduce retention to just two weeks?

Reply:
Done